packages/plugin-schema-validator/src/test/kotlin/ly/iterative/itly/SchemaValidatorPluginTest.kt

```
@@ -69,36 +69,53 @@ class SchemaValidatorPluginTest {
         val itly: Itly = TestUtil.getItly(Options(
             context = Context.VALID_ALL_PROPS,
             plugins = arrayListOf(SchemaValidatorPlugin(Schemas.DEFAULT)),
-            validation = OPTIONS_ERROR_ON_INVALID
+            validation = VALIDATION_OPTIONS_ERROR_ON_INVALID
         ))
 
         val exception = Assertions.assertThrows(IllegalArgumentException::class.java) {
-            itly.validate(invalidEvent)
+            itly.track(null, invalidEvent)
         }
 
+        println("HERE!!!!!")
+        System.err.println("Hello standard error!")
+        println(exception.message)
+
         Assertions.assertEquals(invalidEventExpectedErrorMessage, exception.message)
     }
 
     @Test
-    fun itlyLoad_invalidContextWithErrorOnInvalid_throwsError() {
-        val exception = Assertions.assertThrows(IllegalArgumentException::class.java) {
+    fun itlyLoad_invalidContext_succeeds() {
```

Should throw on invalid right?

---

Nevermind, I see context is now validated with track